common/app/views/main.scala.html

```diff
@@ -40,12 +40,13 @@
             ("has-membership-access-requirement", page.metadata.requiresMembershipAccess),
             ("childrens-books-site", page.metadata.sectionId == "childrens-books-site"),
             ("has-super-sticky-banner", content.exists(_.tags.hasSuperStickyBanner)),
-            ("is-immersive", content.exists(_.content.isImmersive)),
+            ("has-absolute-container", page match {
```

Comment:
minor: since this is actually evaluating if the page is Hosted, maybe we should make the classname reflect that? e.g. `is-hosted-content`. 

We can probably also rely on the page metadata: `page.metadata.isHosted`

Which means this might work:
```
("is-hosted-content", page.metadata.isHosted),
```

Reply:
agree. i wanted to have the class name reflect why it's being applied but really that's what `git blame` is for. WIll change it